configuration/exodus-pipeline.yaml

```diff
@@ -305,7 +316,39 @@ Resources:
     # Create artifact bucket policy for dev pipeline
     Condition: IsDev
     Properties:
-      Bucket: exodus-pipeline-artifacts
+      Bucket: !Sub ${project}-pipeline-artifacts
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action: "s3:*"
+            Principal:
+              Service:
+                - s3.amazonaws.com
+            Resource: !Sub arn:aws:s3:::${project}-pipeline-artifacts
+
+  CloudTrailBucket:
+    Type: AWS::S3::Bucket
+    # Create bucket for cloudtrail logs
+    Condition: IsExodusProd
+    Properties:
+      BucketName: cloudtrail-logs
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      LifecycleConfiguration:
+        Rules:
+          - Status: "Enabled"
+            ExpirationInDays: 1
```

Comment:
The documentation I found was unclear to me; am I correct in assuming this will permenently delete all logs after 24 hours?

Reply:
Correct